docs/ixctl/Sign-Up.md

```diff
@@ -11,3 +11,17 @@
 6. Enter your members / route server information.
 7. Work with FullCtl Admin to onboard / verify information.
 8. Request verification from FullCtl Admin.
+
+## Details
+The free trial of ixCtl allows Users to enter IX and route server information and view how ixCtl functions. When you determine ixCtl is a good fit and meets your needs, contact us at <support@fullctl.com>. We will send you a form to fill out for each IX that will be onboarded. 
+
+FullCtl staff will work with you to develop a plan for migrating from your current system to ixCtl. This process involves working with our Team to prepare for and schedule the route server migrations. We schedule maintenance windows that fit your schedule and guide you through the transition process. 
+
+Cost for ixCtl services:
+<ul>
+<li>Up to 35 members = free</li>
+<li>36-75 members = $200/month</li>
+<li>76+ members = $500/month</li>
+</ul>
```

Comment:
The `<ul>` `<li> `can be removed and replaced with
```

1. Up to 35..
2. 36-75...
...
```
Blank line above 1. is intentional and important @sunbuch